index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Daniels Portfolio</title>
+</head>
+<body>
+    <nav>
+        <ul class="buger">
+
+            <li class="Mi">Daniel</li>
+           <li class="ham">
+            <i class="fa fa-bars"></i> 
+           </li>  
+        </ul>
+    </nav>
+    <section class="homesection">
+        <div>
+<h2 class="intro">I'm Daniel glad to see you</h2>
+<p class="intro1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+</div>
+<div>
+    <p class="social">LET'S CONNECT</p>
+    <ul class="socialicons">
+        <li class="socialicon1"><img src="./img/Vector.svg" alt="twitter logo" /></li>
+        <li class="socialicon1"><img src="./img/WhatsApp Image 2023-06-02 at 13.56.22 (1).jpeg" alt="LinkedIn logo"/></li>
+        <li class="socialicon1"><img src="./img/Vector (3).svg" alt=""/></li>
+        <li class="socialicon1"><img src="./img/Vector (2).svg" alt=""/></li>
+        <li class="socialicon1"><img src="./img/Vector (1).svg" alt=""/></li>
+    </ul>
```

Comment:
Great work so far, but make sure that each icon has a link (`a` tag).
Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. 😄